engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -43,6 +42,7 @@ object SlickDataAccess {
   private lazy val rootDatabaseConfig = rootConfig.getConfig("database")
   private lazy val databaseConfigName = rootDatabaseConfig.getStringOption("config")
   lazy val defaultDatabaseConfig = databaseConfigName.map(getDatabaseConfig).getOrElse(rootDatabaseConfig)
+  val actorSystem = ActorSystem("slick-retry-scheduler")
```

Comment:
I would think we want to share the ActorSystem; might it be possible to pass it as an implicit where required?